requirements-dev.txt

```diff
@@ -18,3 +18,8 @@ pytest-faulthandler<2.0; platform_python_implementation == "PyPy"
 pytest-timeout
 feedparser
 python-hglib
+flake8==4.0.1
+pyflakes==2.4.0
+mccabe==0.6.1
+pycodestyle==2.8.0
+
```

Comment:
Not sure what's the reasoning here, but we want just the latest version of `flake8` (the latest at every time, not the latest now), and no other dependencies. If `flake8` needs those other packages, they'll be installed automatically.

Reply:
Thanks @datapythonista